proto/Cargo.toml

```diff
@@ -27,6 +27,6 @@ protobuf = "2"
 lazy_static = { version = "1.3", optional = true }
 
 [build-dependencies]
-protobuf-build = { version = "0.11", default-features = false }
+protobuf-build = { version = "=0.11.3", default-features = false }
```

Comment:
Is it possible to limiting the grpcio-compiler version?

Reply:
Done